english/bear-story/bear-story.md

```diff
@@ -2,7 +2,7 @@
 Nice to meet you," 
 she says, as she dances around in her bear dress 
 and summons two giant bears to haul you away to 
-another fantasy world as her bear slave. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.